docs/docs/transactions.md

```diff
@@ -67,11 +74,11 @@ CLS works like a thread-local storage for callbacks. What this means in practice
 
 ```js
 sequelize.transaction(function (t1) {
-  namespace.get('transaction') === t1;
+  namespace.get('transaction') === t1; // prints true
```

Comment:
yea you are right. i mean to do `// true`. do you prefer that or you really want to leave it as that? cause it confused me at first look.